_includes/disqus_comments.html

```diff
@@ -0,0 +1,25 @@
+{% if page.comments and jekyll.environment == "production" %}
```

Comment:
:like: I just made that flag optional, meaning that comments will be showed by default